packages/composite-checkout/src/lib/payment-methods/registered-methods.js

```diff
@@ -35,3 +35,16 @@ export default function loadPaymentMethods() {
 		SubmitButtonComponent: () => <button>Pay</button>,
 	} );
 }
+
+export function getAriaLabelForPaymentMethodSelector( methodSlug, localize ) {
+	switch ( methodSlug ) {
+		case 'apple-pay':
+			return localize( 'Check out with Apple Pay' );
+		case 'card':
+			return localize( 'Check out with a credit card' );
+		case 'paypal':
+			return localize( 'Check out with PayPal' );
+		default:
+			throw `Unrecognized payment method slug ${ methodSlug }`;
```

Comment:
While this will work it's best to throw an "instance" of an `Error` object because you get a lot of "free" data (like a stack trace) with the `Error`. Also any `catch` outside your module will expect an object.